### ISSUE TITLE
Support for sails-migrations for versions > 0.10.1

### ISSUE THREAD

**Issue by itayadler** [balderdashy/sails-postgresql#107]
Me and @bluehotdog are trying to fix [sails-migrations](https://github.com/BlueHotDog/sails-migrations) in order to support the new sails versions.

We're finding it difficult to run the migrations as we're required to `register` to the adapter a Waterline.Collection beforehand - This is a requirement that is weird for a migration. If we'll try to use registerConnection a second time it'll fail due a duplication error.

One way that could help us workaround this issue is to provide an updateConnection method that we can use to update later on the migration table (after loading sails with our schema_migration model), or to consider removing the dependency of the collections from the adapter.

We think that the adapters shouldn't hold any state or make any assumptions on the database schema, and should be more "functional", in that it will always goto the database to look for the current state.

**Comment by sailsbot** [comment]
Thanks for posting, @itayadler.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
